### ISSUE TITLE
SHA Sum in the docs doesn't match

### ISSUE THREAD

**Issue by ghost** [BlackArch/blackarch-guide#70]
The SHA sum of the "strap.sh" script does not match the SHA sum provided in the English guide PDF.

**Comment by noraj** [comment]
Seems like this script has been often updated lately but that the hash change was reflected only on the website and not on the PDF documentation.

**Comment by noraj** [comment]
cf.

https://github.com/BlackArch/blackarch-guide/blob/8325d30faa6e1265a01b83579098811af389192c/latex/blackarch-guide-en.tex#L180

https://github.com/BlackArch/blackarch-site/blob/e187f6376d4406de44fd5d0b6fbc1d988930c53b/generators/downloads.html.sh#L157
https://github.com/BlackArch/blackarch-site/blob/e187f6376d4406de44fd5d0b6fbc1d988930c53b/checksums/strap#L1

**Comment by ghost** [comment]
Is it fixed?

**Comment by noraj** [comment]
No, it's not. Would you like to PR on all `blackarch-guide-<lang>.tex` files?
